doc/man3/SSL_stream_conclude.pod

```diff
@@ -0,0 +1,62 @@
+=pod
+
+=head1 NAME
+
+SSL_stream_conclude - conclude the sending part of a QUIC stream
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ __owur int SSL_stream_conclude(SSL *s, uint64_t flags);
```

Comment:
This is a real issue, yes. We should probably consider adding a SSL_write_conclude API or similar, or more likely a SSL_write_flags API which can take a flag CONCLUDE. This can be seen as a performance optimisation however.